src/ansys/acp/core/_tree_objects/model.py

```diff
@@ -437,6 +437,43 @@ def export_shell_composite_definitions(self, path: _PATH) -> None:
                 )
             )
 
+    @supported_since("25.1")
+    def import_materials(
+        self,
+        matml_path: _PATH,
+        *,
+        material_apdl_path: _PATH | None = None,
+    ) -> None:
+        """
+        Import materials from a MatML file.
+
+        Import materials from a ``MatML.xml`` file, and optionally from an associated
+        material APDL file.
```

Comment:
```suggestion
        Import materials from a ``MatML.xml`` (Engineering Data) file. 
        The ``material_apdl_path`` is an optional, pre-generated solver snippet. This 
        is needed in case of variable materials or non-standard material models.
```

Reply:
@roosre slightly reworded your suggestion. For the material APDL file, I guess that's used when exporting the analysis model, correct? Should we mention this here?

Reply:
Yes. It looks like it is only used when exporting a solid model or a surface section cut. In case of a shell model, ACP edits the initial input file where the material definitions are already available.